synapse/lib/snap.py

```diff
@@ -540,14 +517,29 @@ def _getNodeFnib(self, name, valu):
 
             yield node
 
-    async def stor(self, sops):
+    async def stor(self, sops, splices=None):
+        user = '?'
+        if self.user is not None:
```

Comment:
is it even possible for us to have a snap which does not have a user now?

Reply:
nope